test/token-service/test_tokenFeeScheduleUpdateTransaction.js

```diff
@@ -0,0 +1,2701 @@
+          tokenId: fungibleTokenId,
+          customFees: [
+            {
+              feeCollectorAccountId: process.env.OPERATOR_ACCOUNT_ID,
+              feeCollectorsExempt: false,
+              fixedFee: {
+                amount: "-1",
+              },
+            },
+          ],
+          commonTransactionParams: {
+            signers: [fungibleTokenFeeScheduleKey],
+          },
+        });
+        if (response.status === "NOT_IMPLEMENTED") this.skip();
+      } catch (err) {
+        assert.equal(err.data.status, "CUSTOM_FEE_MUST_BE_POSITIVE");
+        return;
+      }
+
+      assert.fail("Should throw an error");
+    });
+
+    it("(#3) Updates a token's fee schedule with a fixed fee with an amount of 9,223,372,036,854,775,807 (int64 max)", async function () {
+      const feeCollectorAccountId = process.env.OPERATOR_ACCOUNT_ID;
+      const feeCollectorsExempt = false;
+      const amount = "9223372036854775807";
+      const response = await JSONRPCRequest("updateTokenFeeSchedule", {
+        tokenId: fungibleTokenId,
+        customFees: [
+          {
+            feeCollectorAccountId: feeCollectorAccountId,
+            feeCollectorsExempt: feeCollectorsExempt,
+            fixedFee: {
+              amount: amount,
+            },
+          },
+        ],
+        commonTransactionParams: {
+          signers: [fungibleTokenFeeScheduleKey],
+        },
+      });
+      if (response.status === "NOT_IMPLEMENTED") this.skip();
+
+      await retryOnError(async () =>
+        verifyTokenFeeScheduleUpdateWithFixedFee(
+          fungibleTokenId,
+          feeCollectorAccountId,
+          feeCollectorsExempt,
+          amount,
+        ),
+      );
+    });
+
+    it("(#4) Updates a token's fee schedule with a fixed fee with an amount of 9,223,372,036,854,775,806 (int64 max - 1)", async function () {
+      const feeCollectorAccountId = process.env.OPERATOR_ACCOUNT_ID;
+      const feeCollectorsExempt = false;
+      const amount = "9223372036854775806";
+      const response = await JSONRPCRequest("updateTokenFeeSchedule", {
+        tokenId: fungibleTokenId,
+        customFees: [
+          {
+            feeCollectorAccountId: feeCollectorAccountId,
+            feeCollectorsExempt: feeCollectorsExempt,
+            fixedFee: {
+              amount: amount,
+            },
+          },
+        ],
+        commonTransactionParams: {
+          signers: [fungibleTokenFeeScheduleKey],
+        },
+      });
+      if (response.status === "NOT_IMPLEMENTED") this.skip();
+
+      await retryOnError(async () =>
+        verifyTokenFeeScheduleUpdateWithFixedFee(
+          fungibleTokenId,
+          feeCollectorAccountId,
+          feeCollectorsExempt,
+          amount,
+        ),
+      );
+    });
+
+    // Bug in services
+    it.skip("(#5) Updates a token's fee schedule with a fixed fee with an amount of -9,223,372,036,854,775,808 (int64 min)", async function () {
```

Comment:
This and other `int64 min` tests are getting skipped, is this the same problem in services as what we saw with `autoRenewPeriod` for `tokenCreate`? Where this value is considered the "unset" value and doesn't properly get parsed as such?

Reply:
[Line 167, CustomFeesValidator](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java#L167), requires that a fixed custom fee _must_ be strictly greater than zero.

Reply:
This _should_ be treating a `0` value as default/unset and returning an error, according to the code [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java#L167).